### ISSUE TITLE
Dataset Split for Linear Evaluation and Fine Tuning

### ISSUE THREAD

**Issue by kuwar81523** [google-research/simclr#184]
Can someone tell me which of the following methods are being used during the evaluation stage :

- Doing pretraining on **datasplit1** and during the evaluation we use **datasplit1** as train and other split as test
- Doing pretraining on **datasplit1** and during the evaluation we use **datasplit2** as train and other split as test

**Comment by chentingpc** [comment]
Both are valid, depends on specific setting. For standard setup, we pretrain on ImageNet and fine-tune / linear eval on ImageNet (eval on their validation set). For transfer learning setup, we pretrain on ImageNet and fine-tune on another dataset (e.g. CIFAR-10 train, and eval on CIFAR-10 test).
